scripts/tasks/sandbox-parts.ts

```diff
@@ -488,7 +490,6 @@ export async function setupVitest(details: TemplateDetails, options: PassedOptio
               name: "chromium",
               provider: "playwright",
               headless: true,
-              screenshotFailures: false,
```

Comment:
Is this related?

Reply:
I'm cleaning up things from changes we made but forgot to clean up. This setting is false by default now